props-core/src/main/java/sh/props/SyncStore.java

```diff
@@ -78,61 +73,6 @@ private static void assertLayerIsValid(Layer search, Pair<String, Layer> vl) {
     return null;
   }
 
-  /**
-   * Convenience method to check the correctness of the specified layer object.
-   *
-   * @param layer a reference that could be null
-   * @return a non-null reference
-   */
-  private static Layer nonNullLayer(@Nullable Layer layer) {
```

Comment:
removed unnecessary code, as now get(..., layerAlias) is handled directly in Registry